django/core/handlers/asgi.py

```diff
@@ -187,7 +188,35 @@ async def handle(self, scope, receive, send):
         if isinstance(response, FileResponse):
             response.block_size = self.chunk_size
         # Send the response.
-        await self.send_response(response, send)
+        try:
+            await self.disconnect_or_send(receive, response, send)
+        except RequestAborted:
```

Comment:
@carltongibson This exception is raised and the function returns None but TimeoutError is not raised to pass the test. The send_response task is pending when the RequestAborted exception is raised but the Timeout Error is never raised even though no response is sent to the client.